website/docs/r/ruleset.html.markdown

```diff
@@ -237,7 +237,9 @@ The following arguments are supported:
 * `increment` - (Optional)
 * `overrides` - (Optional) List of override configurations to apply to the ruleset (refer to the [nested schema](#nestedblock--action-parameters-overrides)).
 * `products` - (Optional) Products to target with the actions. Valid values are `"bic"`, `"hot"`, `"ratelimit"`, `"securityLevel"`, `"uablock"`, `"waf"` or `"zonelockdown"`.
-* `ruleset` - (Optional) Which ruleset to target. Valid value is `"current"`.
+* `ruleset` - (Optional) Which ruleset ID to target.
```

Comment:
cc @pedrosousa

Reply:
i'm going to land this PR (for the sake of needing to cut a breaking change sometime this week) however feel free to raise a PR if this isn't suitable in your own time and we can get this addressed.

Reply:
Sorry, for some reason I missed this one... I'll create a separate PR. 👍🏼